Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.yml

```diff
@@ -2252,7 +2252,1545 @@ script:
     - contextPath: MicrosoftATP.Machine.MachineTags
       description: Set of machine tags
       type: String
-  dockerimage: demisto/crypto:1.0.0.8854
+  - arguments:
+    - default: false
+      defaultValue: '50'
+      description: The maximum amount of indicators to return. Default is 50.
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    deprecated: false
+    description: Lists all indicators by the ID that the system creates when the indicator
+      is ingested.
+    execution: false
+    name: microsoft-atp-list-indicators
+    outputs:
+    - contextPath: MicrosoftATP.Indicators.id
+      type: String
+      description: Created by the system when the indicator is ingested. Generated
+        GUID/unique identifier.
+    - contextPath: MicrosoftATP.Indicators.action
+      type: String
+      description: "The action to apply if the indicator is matched from within the\
```

Comment:
whats with the `\`s ?

Reply:
output of demisto-sdk format.

Reply:
please open an issue for this and remove manually, there are quite a few descriptions where these appear in

Reply:
but it's fine. It just says the line ends with a space.

Reply:
you sure it looks good in the ui? as i said other descriptions have it as well

Reply:
![image](https://user-images.githubusercontent.com/11165655/91975752-8f7c8a80-ed28-11ea-9fbc-f5ec27ecee11.png)
Yeap, it's fine.